packages/commons/src/fhir/transformers/index.ts

```diff
@@ -867,23 +880,7 @@ function createInformantType(
       )
     }
   } else if (context.event === EVENT_TYPE.DEATH) {
-    if (fieldValue === 'MOTHER') {
-      setInformantReference(
-        MOTHER_CODE,
-        MOTHER_TITLE,
-        relatedPersonResource,
-        fhirBundle,
-        context
-      )
-    } else if (fieldValue === 'FATHER') {
-      setInformantReference(
-        FATHER_CODE,
-        FATHER_TITLE,
-        relatedPersonResource,
-        fhirBundle,
-        context
-      )
-    } else if (fieldValue === 'SPOUSE') {
```

Comment:
This fixes #8622

Reply:
![8622](https://github.com/user-attachments/assets/f993acf4-5ebb-4cba-8559-66536cf130d1)